src/System.Linq.Expressions/tests/New/TypeExtensions.cs

```diff
@@ -67,9 +67,14 @@ internal static bool IsByRefParameter(this ParameterInfo pi)
             string name,
             Type[] types)
         {
-            var method = type.GetAnyStaticMethod(name);
-
-            return method.MatchesArgumentTypes(types) ? method : null;
+            foreach (var method in type.GetRuntimeMethods())
```

Comment:
This is different from the original, which was using DeclaredMethods rather than GetRuntimeMethods.  The former won't include inherited members whereas the latter will.  Is this change intentional?